meetings/2023-09-19.md

```diff
@@ -0,0 +1,140 @@
+* WT: If it's based on links I don't have issues with it.
+* MdJ: I have a question on the topic: one way is to read with how do we treat the way we are building specs? another is to design patterns for storing files in pod.
+* TBL: responding to the first: when we have working code, we write down how we do it, use that as examples, analyse it to a shape etc.
+* TBL: If you don't have existing code, e..g bank statements, we go to existing stuff and try to map that.
+* MdJ: I [wrote down](https://pdsinterop.org/conventions/chat/) three chat formats I was aware of two years ago, and I wrote an implementation of one of them storing chat messages in a person's pod. 
+* WT: We are talking specifically about this chat spec after my response to the proposal of chat spec. I would like to look at general patterns for the interop. I want to take opportunity of this use case to look how we can look at such use cases more generally.
+* TBL: To work on two different things is a bit as workin on HTTP and FTP at the same time.
+* MdJ: What we tried to do in the [solid data modules](https://hackmd.io/@michielbdejong/HyIMjmoxn) project is to not having an idea about how an idea of storing data in a pod is good or bad, but just see what is there and what ends up in a pod. Whenever someone wants to write something in a pod, we make the data module compatible with that.
+* TBL: When we talk about how we define c2c specs there's an interesting part of Solid, when you talk about modularity. Typically the modularize around data classes: I can handle an address book, a chat etc. So in this way a chat app signs up for both long chat as 'message' data. 
+* TBL: Suppose I want to link a chat message to a task, in which case it becomes an issue. In other cases, one might take an issue and integrate it in a message... In the spec, we could add some integrations.
+* WT: My main comment is that we can abstract from that and have a general spec which lets us integrate different types of data. The app can also ask pod - do you have this kind of data?
+* TBL: ...
+* WT: One can write chat spec, someone else writes a spec for recipes. We can have general spec which defines how those can be integrated with having dedicated spec for integration between any possible domains.
+
+---
+
+* eP: Coming back a bit to the IRI template with dates, I think we should describe the requirements for that and distinguish it form other popular approaches link [ULIDs](https://github.com/ulid/spec). So what do we want to reach with it? This would also be usable more general.
+* eP: Tomorrow I also want to dmeo shortly the updated Authorization Agent, it also relates to a general way on which way the app knows where it finds data and how it get's access to that specific type of data: a common reference. I will demo how it would work for projects and tasks, and we can then see the general pattern that we would need for any kind of data.
```

Comment:
```suggestion
* eP: Coming back a bit to the IRI template with dates, I think we should describe the requirements for that and distinguish it from other popular approaches like [ULIDs](https://github.com/ulid/spec). So what do we want to reach with it? This would also be more generally usable.
  * ... Tomorrow I also want to briefly demo the updated Authorization Agent. It also relates in a general way to how the app knows where it finds data and how it gets access to that specific type of data: a common reference. I will demo how it would work for projects and tasks, and we can then see the general pattern that we would need for any kind of data.
```